portfolio/src/main/webapp/script.js

```diff
@@ -79,6 +109,55 @@ class Portfolio {
             alert('Error generated in HTTP response from servlet');
         }
     }
+
+    /**
+    * Adds four maps to the page showing four places I really want to visit.
+    */
+    setupPlaceMaps() {
+        let mapOptions;
+
+        for (let loc of Object.entries(this.visitLocs)) {
+            mapOptions = {  
+                            zoom: 8,
+                            center: loc[1]
+                        };
+            this.createMap(mapOptions, loc[0]);
+        }
+
+    }
+
+    /**
+    * Creates a map with pins of my favorite national parks.
+    */
+    setupParkMap() {
+        let usa = { lat: 44.0733586, lng: -97.5443135 };
+        let mapOptions = {
+            zoom: 3,
+            center: usa
+        };
+        let map = this.createMap(mapOptions, 'parks');
+
+        for (let loc of Object.entries(this.parkLocs)) {
+            let name = loc[0], pos = loc[1];
+            let marker = new google.maps.Marker({position: pos, map: map});
```

Comment:
`{foo: foo}` in object literals can be abbreviated as just `{foo}`, so (assuming you change `pos` to `position` as in the previous comment) you can just write `{position, map}`.